Example/Example/AppDelegate.swift

```diff
@@ -34,9 +34,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
             }
         }
 
-        let options = RadarTrackingOptions.responsive
-        options.sync = .all
-        options.showBlueBar = true
+        let options = RadarTrackingOptions.presetContinuous
```

Comment:
I may take another pass at the example app when I have the bandwidth, but this wouldn't be launch-blocking anyways.